dwave/preprocessing/composites/scale.py

```diff
@@ -28,6 +28,9 @@ class ScaleComposite(ComposedSampler):
        sampler (:class:`dimod.Sampler`):
             A dimod sampler.
 
+    .. versionadded:: 0.6.8
+        Support for context manager protocol and :meth:`.close` method.
+
```

Comment:
line 41: `sampler = ScaleComposite(ExactSolver())` opened outside context manager and not closed